src/Translations/TranslationManager.php

```diff
@@ -36,6 +36,127 @@ public function createTranslationFiles(Collection $translations, string $locale)
         }
     }
 
+    public function getPossibleInvalidTranslations() : Collection
+    {
+        $fallbackLocale = config('app.fallback_locale');
+        $appLanguages = collect(config('app.supported_locales'))->reject(fn ($locale) => $locale === $fallbackLocale);
+
+        return $appLanguages->map(function ($locale) use ($fallbackLocale) {
+            $toBecheckedTranslations = collect($this->getTranslations($locale))->dot();
+
+            return collect($this->getTranslations($fallbackLocale))->dot()->filter(
+                function (string $translation, string $key) use ($toBecheckedTranslations) {
+                    $toBecheckedTranslation = $toBecheckedTranslations->get($key);
+
+                    return empty($toBecheckedTranslation)
+                        || (
+                            ($replacements = $this->getReplacementKeys($translation))
+                            && $this->getMissingReplacementKeys(
+                                $this->getReplacementKeys($toBecheckedTranslation),
+                                $replacements
+                            )->isNotEmpty()
+                        );
+                }
+            )->map(function (string $translation, string $key) use ($toBecheckedTranslations, $locale) {
+                return [
+                    'key' => $key,
+                    'locale' => $locale,
+                    'original' => $translation,
+                    'translated' => $translated = $toBecheckedTranslations->get($key),
+                    'missing' => $this->getMissingReplacementKeys(
+                        $this->getReplacementKeys($translated),
+                        $this->getReplacementKeys($translation)
+                    ),
+                ];
+            });
+        })->flatten(1);
+    }
+
+    public function checkPluralization() : Collection
+    {
+        $appLanguages = collect(config('app.supported_locales'));
+
+        $stringVariables = [];
+
+        $appLanguages->each(function ($locale) use (&$stringVariables) {
+            collect($this->getTranslations($locale))
+                ->dot()
+                ->mapWithKeys(function ($translation, $key) {
+                    $matched = preg_match_all('/({\d*}*)|(\|)|(\[\d*,(?:\d+|\**)\])/', $translation, $matches);
```

Comment:
Misschien moeten we deze regex wijzigen zodat deze ongeveer dezelfde is als de regex die gebruikt wordt in Laravel MessageSelector

```suggestion
                    $matched = preg_match_all('/(?:(?:^|\|)[\{\[]([^\[\]\{\}]*)[\}\]]|\|)/', $translation, $matches);
```
In de MessageSelector is het `/^[\{\[]([^\[\]\{\}]*)[\}\]]/` maar die doen vooraf een split op `|` waardoor de conditie altijd in begin van de string staat. En waardoor ze ook geen rekening houden met een `|` die op zichzelf staat